utils/hash.ts

```diff
@@ -1,16 +1,26 @@
 import * as crypto from "crypto";
+import { readableReport } from "italia-ts-commons/lib/reporters";
 import { FamilyMembers } from "../generated/models/FamilyMembers";
+import { FamilyUID } from "../generated/models/FamilyUID";
 
 export const toHash = (s: string): string => {
   const hash = crypto.createHash("sha256");
   hash.update(s);
   return hash.digest("hex");
 };
 
-export const generateFamilyUID = (familyMembers: FamilyMembers): string =>
-  toHash(
-    Array.from(familyMembers)
-      .map(_ => _.fiscalCode)
-      .sort((a, b) => a.localeCompare(b))
-      .join("")
-  );
+export const generateFamilyUID = (familyMembers: FamilyMembers): FamilyUID =>
+  FamilyUID.decode(
+    toHash(
```

Comment:
that would be better to make toHash return NonEmptyString forcing a cast:

`return hash.digest("hex") as NonEmptyString;`